decred/decred/dcr/wire/msgblock.py

```diff
@@ -203,18 +195,17 @@ def btcEncode(self, pver):
         i += extraDataSize
         b[i] = ByteArray(self.stakeVersion, length=uint32).littleEndian()
         i += uint32
-        if i != MaxHeaderSize:
-            raise ValueError(f"unexpected BlockHeader encoded size {i}")
```

Comment:
Forgot to comment on this. I deleted it because it looks like there's no way that check can fail, unless it's a programming error in which case raising an `AssertionError` would be preferable.

And yes, I'm removing that last increment too.

Reply:
I got the same read on it. You're good.